dev/io.openliberty.microprofile.telemetry.1.1.internal/bnd.bnd

```diff
@@ -51,32 +47,34 @@ Service-Component: \
 Import-Package: \
 	io.opentelemetry.api.common,\
 	io.openliberty.microprofile.telemetry.internal.common.resources,\
+	javax.annotation;version="[1.2,2)",\
+	org.eclipse.microprofile.rest.client; version="[1.0.0,3.0.0)",\
+	org.eclipse.microprofile.rest.client.spi; version="[1.0.0,3.0.0)",\
+	org.eclipse.microprofile.config; version="[1.0.0,3.0.0)",\
 	*
 
-Export-Package: \
-	io.openliberty.microprofile.telemetry11.internal.cdi;version='1.0',\
-	io.openliberty.microprofile.telemetry11.internal.rest;version='1.0';thread-context=true
-	
 Private-Package: io.openliberty.microprofile.telemetry11.internal.helper,\
 	io.openliberty.microprofile.telemetry11.internal.sdk
 
 -buildpath: \
-    io.openliberty.jakarta.restfulWS.3.1;version=latest,\
-    io.openliberty.jakarta.annotation.2.1;version=latest,\
-    io.openliberty.jakarta.interceptor.2.1;version=latest,\
-    io.openliberty.jakarta.cdi.4.0;version=latest,\
-    io.openliberty.jakarta.servlet.6.0;version=latest,\
-    com.ibm.ws.org.osgi.annotation.versioning;version=latest, \
-    com.ibm.ws.logging.core;version=latest,\
-    com.ibm.ws.cdi.interfaces.jakarta;version=latest,\
-    io.openliberty.microprofile.telemetry.internal.common;version=latest,\
-    com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
-    io.openliberty.io.opentelemetry.1.29;version=latest,\
-    io.openliberty.org.eclipse.microprofile.config.3.0,\
-    io.openliberty.org.eclipse.microprofile.rest.client.3.0,\
-    io.openliberty.org.jboss.resteasy.common.ee10;version=latest,\
-    com.ibm.websphere.org.osgi.core;version=latest,\
-    com.ibm.ws.container.service;version=latest,\
-    com.ibm.ws.kernel.boot.core;version=latest
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	com.ibm.ws.logging.core;version=latest,\
+	com.ibm.ws.cdi.interfaces;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.config.1.3,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.ws.kernel.boot.core;version=latest,\
+	com.ibm.websphere.javaee.jaxrs.2.0,\
+	com.ibm.websphere.javaee.servlet.3.1,\
+	com.ibm.websphere.javaee.annotation.1.2,\
+	com.ibm.websphere.javaee.interceptor.1.2,\
+	com.ibm.websphere.javaee.cdi.1.2,\
+	com.ibm.websphere.org.eclipse.microprofile.rest.client.1.1;version=latest,\
+	io.openliberty.org.jboss.resteasy.common;version=latest,\
+	io.openliberty.microprofile.telemetry.internal.common;version=latest,\
+	io.openliberty.io.opentelemetry.1.29;version=latest,\
+	io.openliberty.org.jboss.resteasy.common,\
+	org.jsr-305,\
```

Comment:
Not clear to me why JSR 305 (@NonNull) is needed here

Reply:
It looks like we only have it because we're implement OTel interfaces and they use it.

We don't need to add `@Nullable` ourselves and we don't have any static analysis tools which would check it.

At runtime we get away with this only because `javax.annotation` is exported by the annotations feature so the bundle package import is satisfied even though the actual `Nullable` class isn't in our version of that package. The missing `Nullable` class is ignored because it's an annotation.

I think we should remove this classpath entry and the use of `@Nullable`